service/history/workflow/test_util.go

```diff
@@ -26,8 +26,13 @@ package workflow
 
 import (
 	"context"
+	"fmt"
+	"testing"
 	"time"
 
+	gomock "github.com/golang/mock/gomock"
```

Comment:
```suggestion
	"github.com/golang/mock/gomock"
```

Reply:
Weird. :man_shrugging:

Reply:
Oh, it's because vim-go sees that that's how the gomock pkg is imported everywhere else it's used in the workflow package.